### ISSUE TITLE
Extract method resulting in bundles that don't work in IE11 - Need Polyfills?

### ISSUE THREAD

**Issue by pjhartin** [JeffreyWay/laravel-mix#1711]
- Laravel Mix Version: 2.1.11
- Node Version: 8.11.3
- NPM Version 5.6.0
- OS: Ubuntu 18.04

### Description:
When using the[ `.extract()` ](https://github.com/JeffreyWay/laravel-mix/blob/master/docs/extract.md)function to code split into an `app.js` and `vendor.js` bundle, the splitting works perfectly. Adding manifest, vendor and app as scripts to the template works as expected on Chrome, Firefox and Safari, however, in IE11 it always fails to work with manifest.js failing.

It always fails when trying to execute the module function:
`modules[moduleId].call(modules.exports, module, module.exports, __webpack_require__);`
>"Unable to get property 'call' of undefined or null reference.
![image](https://user-images.githubusercontent.com/1385026/42562069-21422394-853e-11e8-8046-02cc2fc76b2a.png)

My initial thought was that this is a polyfill issue, and I have imported "babel-polyfill" into my `entry.js` with:
```
import "babel-polyfill";
```
Importing the above fixes standard polyfill issues on IE when I build without using `.extract()`, however, it never works along side it. 

I now think that the polyfill needs to somehow be bundled with the actual manifest in order for the module loading to work on IE11. 

Does anyone know how to do this?

### Steps To Reproduce:

- Simply add any number of libraries to your project.
- Import some into your entry.js
- Use Laravel mix to extract out some `node_modules` packages using the `extract()` function :

```
let vendorPackages = [
  "vue",
  "vue-carousel",
];

mix
  .js(["src/js/base.js"], "js/app.js")
  .extract(vendorPackages)
```

- Add the `manifest.js`, `vendor.js` and `app.js` to your index.html
- Test on a modern browser (e.g. Chrome) and see that it works
- Test on IE11 and see it fall over

**Comment by ankurk91** [comment]
I never faced this issue.

You need to give us more insights. Create a minimal reproduction and share on a github repo.

**Comment by pjhartin** [comment]
@ankurk91 - I actually built a seperate repo to demonstrate the problem, but I couldn't replicate it.
So I started to individually import the packages used in our current build (the one with the issue) and as it turns out 2 community vue packages were causing the issue. I still do not understand why, but I have removed them from the vendors bundle, and it now works on IE11.

Sorry for raising this issue.
